sqlalchemy_jsonapi/declarative/serializer.py

```diff
@@ -0,0 +1,82 @@
+"""A serializer for serializing SQLAlchemy models to JSON API spec."""
+
+
+class JSONAPISerializer(object):
+    """A JSON API serializer that serializes SQLAlchemy models."""
+
+    def serialize(self, resources):
+        """Serialize resource(s) according to JSON API spec.
+
+        Args:
+            resources: SQLAlchemy model or SQLAlchemy query
+        """
+        if 'id' not in self.fields:
```

Comment:
This is a check that could potentially be moved to `__init__`. 
I am requiring an 'id' right now and as we move further along, when we implement a translation this would then support people who did not necessarily want their model's primary key to be named 'id'.